.github/workflows/algolia/configure-algolia-api.py

```diff
@@ -9,6 +9,14 @@
 index_name = os.environ["ALGOLIA_INDEX"]
 
 
+ONE_WAY_SYNONYMS = {
+    # A list of search terms that have (historically) not returned results
+    # that we can map to existing search terms that we know are good
+    "md5": ["hashbytes"],
+    "fetchdf": ["to_pandas", "to_polars", "to_pyarrow"],
```

Comment:
Is this a place we would want to handle better routing of "unique" -> "distinct"? Or is that a docstring change in the `distinct` doc to increase the search relevance there?

Reply:
This is a place to add that, so if someone searches `unique` the top hit would be `distinct`.  I'll add that.

Reply:
oh, actually, let me test this because I want to see if this just adds `distinct` or if it eliminates the valid `unique` on Arrays

Reply:
Awwwww yeaaahhhh

![2024-09-18_12-20](https://github.com/user-attachments/assets/72c5e415-244d-48b5-b37c-85ab022ab534)